src/main/scala/sbtdatabricks/DatabricksPlugin.scala

```diff
@@ -25,6 +27,10 @@ object DatabricksPlugin extends AutoPlugin {
     val dbcLibraryPath = settingKey[String]("Where in the workspace to add the libraries.")
     val dbcListClusters = taskKey[Unit]("List all available clusters and their states.")
     val dbcRestartClusters = taskKey[Unit]("Restart the given clusters.")
+    val dbcExecuteCommand = taskKey[Seq[CommandStatus]]("Execute a command on a particular cluster")
+    // TODO : Change to a Path / File type for dbcCommandFile
+    val dbcCommandFile = settingKey[String]("Location of file containing the command to be executed")
```

Comment:
Can we make this a `File` right now? Then we can show how in SBT `/` is a macro, and specifying a path like `Path.userHome / "myFolder" / "myFile.py"` will work in both windows and Mac.